src/test/resources/manual-test/readme.md

```diff
@@ -0,0 +1,14 @@
+JSON files for manual tests
+---------------------------
+
+* Launch a virtual machine with `vagrant up` in the root of the project
+* Create a deposit, for details see `docs/api.html`
+* Use the returned UUID to replace XXX in the following command.
+
+    curl -i  -H 'Content-Type: text/plain'  --data-binary "@XXX.json" -X POST -u user001:user001 'http://test.dans.knaw.nl:20190/deposit/XXX/datasetmetadata'
+
+
+Purpose of HTML files
+---------------------
+
+Examine cookies with developer tools of a browser without a fully dressed web client.
```

Comment:
A couple more words on the following things would be helpful:
1. the purposes of each individual HTML file
2. what you can test with them?
3. what steps do you want me to take while testing?
4. what you should expect to find with them?

_future you thanks you on behalf of me and all other developers_ 😉